localization/Xamarin.Localization.Mlaunch/Base.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Xamarin.Hosting")]
+[assembly: InternalsVisibleTo("mlaunch")]
+namespace Xamarin.Localization.Mlaunch
+{
+	public class Base
+	{
+		public Base()
+		{
+		}
+	}
+}
```

Comment:
is the type really needed ?

if so then make it internal, not public.

Reply:
It was autogenerated. I removed it.